source/globals.d.ts

```diff
@@ -1,2 +1,92 @@
 // TODO: Type anything that is of type AnyObject
 type AnyObject = Record<string, any>;
+
+// TODO: Move types to tiny-version-compare repo
+declare module 'tiny-version-compare' {
+	function compareVersions(versionA: string, versionB: string): number
+
+	export = compareVersions
+}
+
+// TODO: Move types to https://github.com/sindresorhus/linkify-urls repository.
+declare module 'linkify-urls' {
+	type Options = {
+		user: string;
+		repo: string;
+		type: string;
+		baseUrl: string;
+		attributes: {
+			rel: string;
+			class: string;
+		};
+	};
+
+	function linkifyUrls(input: string, options: Options): DocumentFragment;
+
+	export = linkifyUrls
+}
+
+// TODO: Move types to the https://github.com/sindresorhus/linkify-issues repository.
+declare module 'linkify-issues' {
```

Comment:
`linkify-issues` and `linkify-urls` now ship with their own TS types, so these can be removed.

Reply:
Sounds good. I'll remove the types.